shell.nix

```diff
@@ -1,4 +1,9 @@
-with import <nixpkgs> { };
+let
+  moz_overlay = import (
+    builtins.fetchTarball "https://github.com/mozilla/nixpkgs-mozilla/archive/master.tar.gz"
```

Comment:
What's this used for?

Reply:
this is for using rust version and related toolchains based on `rust-toolchain.toml` (https://github.com/versotile-org/verso/pull/169/files#diff-e53dfbfffe62ae3c0b411b3938ccffa9fb6a2ecc565f55785ef8daa756631a6bR26) instead of rustup in the package repository. This way we only have set it once, and nix-shell will update with the project simultaneously.
Not a mandatory thing to have, but probably could prevent some future issues.

Reply:
Let's stick to nixokg's rustup until we encounter the future issues.

Reply:
nix conf modifications are removed now.